Consul/Connect.cs

```diff
@@ -328,6 +328,35 @@ public Task<WriteResult<bool>> UpsertIntentionsByName(ServiceIntention intention
         {
             return UpsertIntentionsByName(intention, WriteOptions.Default, ct);
         }
+
+        /// <summary>
+        /// Deletes a specific intention by its unique source and destination.
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="destination"></param>
+        /// <param name="q"></param>
+        /// <param name="ct"></param>
+        /// <returns>A Write Option</returns>
+        public Task<WriteResult> DeleteIntentionByName(string source, string destination, WriteOptions q, CancellationToken ct = default)
+        {
+            var req = _client.Delete("v1/connect/intentions/exact", q);
+            req.Params["source"] = source;
+            req.Params["destination"] = destination;
+            var res = req.Execute(ct);
```

Comment:
There is no need for a temporary variable here, we can return the result outright:
```suggestion
            return req.Execute(ct);
```